### PR TITLE
Update README.adoc

### PR DESCRIPTION
**Pull request by nateziemann**
Added an introduction that description what Kabanero provides and how it relates to Tekton, pipelines, etc.  My review is complete.  I believe Alan has technical concerns that should be addressed separately from this PR.

### CODE REVIEW

**Review by anneqm — APPROVED** [review]
Thank you. Changes look good and enhance the guide.

**Review by dewan-ahmed — COMMENTED** [review]
"...declaring CI/CD-style pipelines" --> declaring CI/CD pipelines

File: README.adoc [hunk 1]
```diff
@@ -17,13 +17,17 @@ permalink: /guides/jenkins-integration-with-kabanero/
...
-:page-releasedate: 2019-10-24
+:page-releasedate: 2019-11-13
 :page-description: Learn how to use Tekton to build an image and how to use Jenkins to deploy an application that uses that image.
-:page-tags: ['Tekton', 'Jenkins', 'CI/CD', 'Collection']
+:page-tags: ['Tekton', 'Jenkins', 'CI/CD', 'Pipeline', 'Kabanero']
 :page-guide-category: collections
 = Jenkins integration with Kabanero
 
-Learn how to use Tekton to build an image and how to use Jenkins to deploy an application that uses that image.
+Kabanero provides a modern DevOps toolchain with pre-built pipelines using an open source project called Tekton. The Tekton project provides Kubernetes-style resources for declaring CI/CD-style pipelines. While it is advantagous to run both CI and CD using Tekton to leverage the power of Kubernetes to manage your toolchain, some may want to reuse Jenkins assets or skills.
```
Comment (Charlotte-Holt) [inline README.adoc:26]: Suggestion:
"Kabanero delivers a modern DevOps toolchain with pre-built Tekton pipelines. The Tekton open source project provides Kubernetes-style resources for declaring CI/CD-style pipelines. While it is advantageous to run both CI and CD with Tekton to leverage the power of Kubernetes, some might want to reuse Jenkins assets or skills."

File: README.adoc [hunk 2]
```diff
@@ -17,13 +17,17 @@ permalink: /guides/jenkins-integration-with-kabanero/
...
+
+With Kabanero, you can customize the deploy section of the pipeline to use Jenkins.
```
Comment (Charlotte-Holt) [inline README.adoc:28]: Suggestion:
"With Kabanero, you can customize the deployment section of the pipeline to use Jenkins."

File: README.adoc [hunk 3]
```diff
@@ -17,13 +17,17 @@ permalink: /guides/jenkins-integration-with-kabanero/
...
+
+In this guide you will learn how to use Tekton to build an image and how to use Jenkins to deploy an application that uses that image.
```
Comment (Charlotte-Holt) [inline README.adoc:30]: Suggestion:
"In this guide, you will learn how to use Tekton to build an image and how to use Jenkins to deploy an application that uses that image."

**Review by Charlotte-Holt — COMMENTED** [review]
@nateziemann @anneqm I added just 3 quick suggestions. Let me know if you have any questions. Thanks!

File: README.adoc (hunk 1 above)
Comment (dewan-ahmed) [reply README.adoc:26]: LGTM

**Review by Charlotte-Holt — APPROVED** [review]

**Pull request merged**